container/common/helpers.go

```diff
@@ -226,7 +203,10 @@ func readString(dirpath string, file string) string {
 
 func readUInt64(dirpath string, file string) uint64 {
 	out := readString(dirpath, file)
-	if out == "" || out == "max" {
+	if out == "max" {
+		return math.MaxUint64
```

Comment:
nit: The max value for cgroup v1 is 9223372036854771712 (max signed integer rounded to nearest page). So to be correct it should be something like this I guess:

```suggestion
		return math.MaxInt64^0xFFF
```

Reply:
I am fine with changing it, but would it be a problem with cgroup v1?  `"max"` is used only on cgroup v2

Reply:
I don't remember where this value is used, but I just thought it was nice with the same value for both cgroup v2. If you try to write `MaxInt64^0xFFF  + 0x1` you get `max` on cgroup v2, and `9223372036854771712` on v1. 9223372036854771712 is also default on v1.

Reply:
Edit: This is used for more than the memory limits, and my above comments only apply to those. Hmm